packages/migrate/migrations/sveltekit-2/migrate.spec.js

```diff
@@ -30,3 +31,22 @@ for (const sample of read_samples(new URL('./tsjs-samples.md', import.meta.url))
 		assert.equal(actual, sample.after);
 	});
 }
+
+test('Does not downgrade versions', () => {
```

Comment:
I think it would be better to have a general test in `utils.spec.js`, since this isn't specific to the SvelteKit 2 migration